user-guide/getting-started.md

```diff
@@ -62,9 +62,9 @@ The Edge Policy Console must authenticate your session using a Kubernetes Secret
 
 For more information, see [Edge Control](../../reference/edge-control).
 
-## Configure TLS for Automatic HTTPS
+## Configure TLS Termination and Automatic HTTPS
 
-If you have the ability to update your DNS, Ambassador can automatically configure a valid TLS certificate for you, eliminating the TLS warning. If you do not have the ability to update your DNS, skip to the next section, "Create a Mapping."
+**The Ambassador Edge Stack enables TLS termination by default using a self-signed certificate.** If you have the ability to update your DNS, Ambassador can automatically configure a valid TLS certificate for you, eliminating the TLS warning. If you do not have the ability to update your DNS, skip to the next section, "Create a Mapping."
```

Comment:
Sorry, could you add a link to the Host CRD so they have somewhere to go to disable TLS

Reply:
okay added link!